src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -59,25 +64,26 @@ class NotificationService(
         log.info("Finished sending notifications")
     }
 
+    private fun generateAndSaveNotifications() {
+        val allPrisons = prisonService.getAllPrisons().distinctBy { it.csrPlanUnit }
+        val newShiftNotifications = allPrisons
+                .flatMap { prison ->
+                    csrClient.getShiftNotifications(prison.csrPlanUnit, prison.region)
+                            .map {
+                                if (it.actionType == ShiftActionType.EDIT.value && checkIfNotificationsExist(it.quantumId, it.shiftDate, it.shiftType)) {
```

Comment:
These now use the string values rather than parsing them to Enum, before we were doing (in js) Enum === String